Quick start doc in README

TylerLeonhardt opened the pull request:
fixes #1057

Note, this is not exhaustive but I wanted to write _something_ down to at least give folks a place to start.

In theory a more exhaustive doc should be elsewhere (like on docs.microsoft.com) but this will do for now.

SteveL-MSFT commented on README.md:
```diff
@@ -29,6 +29,76 @@ The functionality in PowerShell Editor Services is already available in the foll
+Named Pipes are the way to go.
+They give you all the benefit of the Lanaguge Server Protocol with extra capabilities that you can take advantage of:
+
+- The PowerShell Integrated Console
```
Lists need to have 2 spaces indentation

TylerLeonhardt commented:
I agree with what you're saying but I think a lot of what you're suggesting should be in a more exhaustive doc instead of a quick start that this is trying to be.

TylerLeonhardt commented:
> Why Stdio doesn't support the Integrated Console

Added

> What startup looks like and what the timing is between client and server (so people don't think the script has hung)

Added

> Security considerations for clients using named pipes

Should be in a different doc

> How shutdown works and the lifetime of the server compared to the client

Should be in a different doc

> What we consider to be stable and what clients can depend on (messages vs linking)

Not needed at the moment

> Why debugging with PowerShell is a bit different (i.e. happens in the same process)

Added

rjmholt commented on README.md:
```diff
@@ -67,10 +157,10 @@ contribute code, documentation, tests, or bug reports, please read our [Contribu
+-  [Keith Hill](https://github.com/rkeithhill) - [@r_keith_hill](http://twitter.com/r_keith_hill)
+-  [Patrick Meinecke](https://github.com/SeeminglyScience) - [@SeeminglyScienc](http://twitter.com/SeeminglyScienc)
+-  [Tyler Leonhardt](https://github.com/tylerleonhardt) - [@TylerLeonhardt](http://twitter.com/tylerleonhardt)
+-  [Rob Holt](https://github.com/rjmholt) - no twitter
```
May as well

```suggestion
-  [Rob Holt](https://github.com/rjmholt) - [@rjmholt](https://twitter.com/rjmholt)
```
